### PR TITLE
Prepare DWDS `16.0.2+1` release

### PR DESCRIPTION
**Pull request by elliette**
Work towards https://github.com/flutter/flutter/issues/119084

FYI @christopherfujino

### CODE REVIEW

**Review by christopherfujino — APPROVED** [review]
LGTM

File: dwds/pubspec.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: dwds
 # Every time this changes you need to run `dart run build_runner build`.
-version: 16.0.2
+version: 16.0.2+1
```
Comment (annagrin) [inline dwds/pubspec.yaml:3]: Do we need to change the version in the changelog as well?

**Review by annagrin — APPROVED** [review]
LGTM with a small request

File: dwds/pubspec.yaml (hunk 1 above)
Comment (elliette) [reply dwds/pubspec.yaml:3]: Nope, the changelog was updated in https://github.com/dart-lang/webdev/pull/1930

**Pull request merged**